inst/components/scss/nav_spacer.scss

```diff
@@ -1,5 +1,11 @@
 /* CSS behind nav_spacer() */
+
 @mixin nav-spacer() {
+  .card-header {
+    > .bslib-nav-spacer {
+        margin-left: auto !important;
+    }
+  }
```

Comment:
I think this rule would be better located in `card.scss` under the new `.card-header` rules, along with a comment about why we're including it. Also, we should only include `!important` in extreme cases where it's absolutely necessary.